### ISSUE TITLE
wrong note is shown up after deleging a note

### ISSUE THREAD

**Issue by jakobroehrl** [nextcloud/notes#1098]
## Steps to reproduce
1. click on a note to show the content
2. delete another note with the ... menu > Delete not
3. click on some notes, now the wrong notes are shown up
4. solve this with a page reload

**Comment by marxistvegan** [comment]
This has happened to me as well both on 27.0.1 and also on 27.0.2 both running notes 4.81
I rely on notes daily and needed to find a secondary approach to this until this gets resolved
This is also related to [new ones being immediately deleted](#1107) as well and also related to [notes being locked](#981).

**Comment by tuxick** [comment]
Same here

**Comment by joshtrichards** [comment]
> solve this with a page reload

This may have been the same as #1298 and addressed in #1322. Can you confirm whether this behavior is gone for you in v4.10.1?

**Comment by MrRinkana** [comment]
I don't believe those are the same issue, either way I believe I'm having this issue on 4.11.0, nc 29.0.9.

 For me after deleting a note the same note is shown independent of what note i select.

See behaviour [here](https://github.com/user-attachments/assets/b829c57a-d1f6-4c81-aa85-92e7d8bd9132)

Steps to reproduce:
1. Browse around notes, things work as expected.
2. Delete a note that you no longer need (start of video)
3. Select another note (this works)
4. Try selecting any other note, or category (does **not work**, previous note is shown)
5. Reload
6. The note you selected last appears (things work again)

If one stays at the note that was deleted, a message appears: "session for editing note has expired, please reload page", maybe that breaks the UI?
